hazelcast/src/main/java/com/hazelcast/config/NearCacheConfig.java

```diff
@@ -164,31 +163,32 @@ public NearCacheConfig(int timeToLiveSeconds, int maxSize, String evictionPolicy
     }
 
     public NearCacheConfig(NearCacheConfig config) {
-        name = config.getName();
-        evictionPolicy = config.getEvictionPolicy();
-        inMemoryFormat = config.getInMemoryFormat();
-        invalidateOnChange = config.isInvalidateOnChange();
-        maxIdleSeconds = config.getMaxIdleSeconds();
-        maxSize = config.getMaxSize();
-        timeToLiveSeconds = config.getTimeToLiveSeconds();
-        cacheLocalEntries = config.isCacheLocalEntries();
-        localUpdatePolicy = config.localUpdatePolicy;
+        name = config.name;
```

Comment:
Change the copy constructor to use fields instead of getters. This reduces the amount of code and doesn't change the logic.